src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Wysiwyg.Edit.cshtml

```diff
@@ -7,7 +7,9 @@
     var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);
 }
 
+<style asp-name="trumbowyg-plugins" version="2"></style>
 <style asp-name="trumbowyg" version="2"></style>
+<script asp-name="trumbowyg-plugins" version="2" at="Foot"></script>
```

Comment:
Do we need the version specified here? I'd do that in a consumer app, but here, we'd need to update these templates if we update the package and that requires us to adapt our code.

Also for the `HtmlBodyPart` editor.

Reply:
I didn't want to change the version in this PR, even though I'm not sure we need that at all.

Reply:
I mean, you added these lines. I'm saying that these can be just `<style asp-name="trumbowyg-plugins"></style>` and `<script asp-name="trumbowyg-plugins" at="Foot"></script>".

Reply:
Oh yes, sorry. Those must be in line with the editor itself, which was already set to version 2 explicitly.